yffi/src/lib.rs

```diff
@@ -2177,7 +2177,7 @@ pub unsafe extern "C" fn yxmltext_insert_embed(
         .expect("provided transaction was not writeable");
     let txt = XmlTextRef::from_raw_branch(txt);
     let index = index as u32;
-    let content: Any = content.read().into();
```

Comment:
The same issue existed in inserting embed elements into text: we couldn't insert other Y types as text embeds.